.github/workflows/no-response.yml

```diff
@@ -0,0 +1,37 @@
+name: No response
+
+# `issues`.`closed`, `issue_comment`.`created`, and `scheduled` event types are required for this Action
+# to work properly.
+on:
+  issues:
+    types: [closed]
+  issue_comment:
+    types: [created]
+  schedule:
+    # These runs in our repos are spread evenly throughout the day to avoid hitting rate limits.
+    # If you change this schedule, consider changing the remaining repositories as well.
+    # Runs at 12 am, 12 pm
+    - cron: '0 0,12 * * *'
```

Comment:
These run times should be different per repository so we avoid running into API call limits on Github.